main.go

```diff
@@ -49,15 +66,16 @@ func main() {
 		}()
 	}
 
+	router := NewRouter()
 	svr := &http.Server{
-		Addr:         port,
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 10 * time.Second,
+		Addr:          ":" + *port,
```

Comment:
I find `fmt.Sprintf(":%s",*port)` a bit more precise instead of concatenating as you can easily see and control the output formatting, but that's just my opinion